packages/devtools/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.9.2
+* Update `package:vm_service` to `^8.1.0` [#3545](https://github.com/flutter/devtools/pull/3545)
+
 ## 2.9.2-dev.1
```

Comment:
I think you can remove `2.9.2-dev.1` now and move everything there under `2.9.2`

Reply:
Done!